aten/src/TH/generic/THTensorMath.c

```diff
@@ -3040,16 +3003,11 @@ LAB_IMPLEMENT_BASIC_FUNCTION(abs,abs)
 
 #if defined(TH_REAL_IS_BYTE)
 
-#define TENSOR_IMPLEMENT_LOGICAL_SUM(NAME, OP, INIT_VALUE) \
-  int THTensor_(NAME)(THTensor *tensor) \
-  { \
-    int sum = INIT_VALUE;                               \
-    TH_TENSOR_APPLY(real, tensor, sum = sum OP *tensor_data;); \
-    return sum; \
-  }
+TENSOR_IMPLEMENT_ACCALL(logicalallall, int, &&, 1)
```

Comment:
Can we rename them to `logicalAndAll` and `logicalAnd` (for the one that reduces over a dimension)?